docs/sources/mimir/send/otel-exponential-histograms/_index.md

```diff
@@ -0,0 +1,117 @@
+---
+description: Learn how to collect and send exponential histograms with the OpenTelemetry Collector
+keywords:
+  - send metrics
+  - exponential histogram
+  - OpenTelemetry
+  - instrumentation
+menuTitle: OpenTelemetry exponential histograms
+title: Send OpenTelemetry exponential histograms to Mimir
+weight: 200
+---
+
+# Send OpenTelemetry exponential histograms to Mimir
+
+You can collect and send exponential histograms to Mimir with the OpenTelemetry Collector. OpenTelemetry [exponential histograms](https://opentelemetry.io/docs/specs/otel/metrics/data-model/#exponentialhistogram) are compatible with Prometheus native histograms. The key difference is that exponential histograms store the `min` and `max` observation values explicitly, whereas native histograms don't. This means that for exponential histograms, you don't need to estimate these values using the 0.0 and 1.0 quantiles.
+
+The OpenTelemetry Collector supports collecting exponential histograms and other compatible data formats, including native histograms and Datadog sketches, through its receivers and sending them through its exporters.
+
+{{< admonition type="note" >}}
+The availability of different receivers and exporters depends on your OpenTelemetry Collector [distribution](https://opentelemetry.io/docs/concepts/distributions/).
+{{< /admonition >}}
+
+You can use the OpenTelemetry protocol (OTLP) to send exponential histograms to Grafana Mimir in their existing format, or you can use the Prometheus remote write protocol to send them as Prometheus native histograms. Refer to [Send native histograms to Mimir](https://grafana.com/docs/mimir/<MIMIR_VERSION>/send/native-histograms/).
```

Comment:
We did not document sending with OTEL collector, so let's remove the reference. Also I realized when looking into this that there's two OTLP exporters and we only support the one over HTTP protocol.

```suggestion
You can use the OpenTelemetry protocol (OTLP) over HTTP to send exponential histograms to Grafana Mimir in their existing format, or you can use the Prometheus remote write protocol to send them as Prometheus native histograms.
```

We could refer to the the OTLP HTTP exporter and Promtheus remote write exporter documentation:
- https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/otlphttpexporter/README.md
- https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/prometheusremotewriteexporter/README.md